src/Extensions/Array.test.ts

```diff
@@ -0,0 +1,36 @@
+import { describe } from 'riteway'
+import './Array'
+
+describe('Array', async (should: any) => {
+  const { assert } = should('')
+
+  {
+    const arrayWithObject = [
+      ['key1', 'value1'],
+      ['key2', 'value2']
+    ]
+
+    const result = arrayWithObject.toObject()
+
+    assert({
+      given: 'array with two pairs of key values',
+      should: 'transform to an object',
+      actual: typeof result === 'object',
+      expected: true
+    })
+
+    assert({
+      given: 'array with two pairs of key values',
+      should: 'get the keys',
```

Comment:
the object should have the same keys as the array?

Reply:
do you disagree with the `should` text?

Reply:
yeah, sorry. I was suggesting a text for the `should`.